Makefile

```diff
@@ -54,15 +54,6 @@ tools:
 deps: tools
 	$(GO_GET) -v -t -d ./...
 
-.PHONY: gen-codes
-gen-codes:
-	$(PROTOC) --version || sh tools/install_protoc.sh
-	/bin/sh tools/protocol_gen.sh
-
-.PHONY: gen-docs
-gen-docs: build
-	rm -rf $(PLUGIN_DOC_DIR)
-	$(OUT_DIR)/$(BINARY)-$(VERSION)-$(OSNAME)-$(ARCH) docs --output=$(PLUGIN_DOC_DIR)
```

Comment:
I'm not sure why you remove these 2 targets, if you do, please also remove the related files, such as `tools/protocol_gen.sh`, as it's not used and it doesn't work now since you modify the package name.

Reply:
One file wouldn't be able to be removed, satellite has its own event proto. But we should remove most of others, and host the generated source codes directly to avoid the complexity.